scripts/globals/effects/confrontation.lua

```diff
@@ -5,16 +5,20 @@
 -----------------------------------
 
 function onEffectGain(target,effect)
-    if (target:getPet()) then
+    if target:getPet() then
         target:getPet():addStatusEffect(effect)
     end
+
+    if target:getObjType() == tpz.objType.PC then
+        target:clearTrusts()
+    end
```

Comment:
Does this forcible despawn happen when the status is granted?

What fights does this apply to?

Reply:
I was just going on the wikis:
https://ffxiclopedia.fandom.com/wiki/Confrontation

Reply:
Wiki feels off. Are trusts despawned when spawning a Dynamis zone boss?

@project-topaz/researcher if anyone feels like running in and popping one

Reply:
Vid from cloudchief as of the 5th of June says no
https://youtu.be/97QYZQxY9dA?t=2331